### ISSUE TITLE
(theme) Extension Search input box has inconsistent transparency on text selection background

### ISSUE THREAD

**Issue by ZYinMD** [microsoft/vscode#75012]
(It is already described in #57281, but I think this one better identifies the bug, and may have found the fix) @roblourens 

The text selection background color in all input boxes / search boxes are determined by `selection.background` in [Theme Color Reference](https://code.visualstudio.com/api/references/theme-color), including the search box where you search for extensions. However, in said search box, the selection background has a lower opacity

Steps to Reproduce:
1. Type something in the search box in the Extensions sidebar, select the text you just typed
2. Observe how it's more transparent than all other input boxes.
3. Change `selection.background` in your color theme, repeat 1 and 2.

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Yeah I started looking at it in that issue, later tweaked it with https://github.com/microsoft/vscode/commit/f40ca995d51e2b47e78e246b4e361a2b1b2d3e35 but it's not perfect. The reason is that we are trying to imitate the native input box selection color behavior in a non-native input box (actually a small editor instance). In our custom input box we can set the background color to be exactly whatever we want, but if you set the selection color in a native input box, it actually makes it slightly transparent with a blend mode that I couldn't figure out how to replicate exactly. I decided it was close enough for some common themes at a 40% transparency. If you have any idea how to be more accurate, I am open to PRs.

**Comment by ZYinMD** [comment]
Hi thanks for the quick response. Sorry I won't be much of a help, I just discovered it when building a theme - I used an already-very-transparent color on `selection.background`, then found it to be nearly invisible in that search box. 

One ugly way to fix could be to create a new variable for native input boxes, and set it to be the same as `selection.background` if undefined, so theme developers can use it if they want to. But it's probably not worth it.

If you want to figure out the blend mode, maybe do a screenshot, then use a color picker to get the rendered value, then [do some math](https://stackoverflow.com/questions/12228548/finding-equivalent-color-with-opacity), just saying 😊
